sdk/python/kfp/compiler/compiler.py

```diff
@@ -1146,7 +1146,7 @@ def _validate_workflow(workflow: dict):
             container:
               image: docker/whalesay:latest""")
     except:
-      warnings.warn("Cannot validate the compiled workflow. Found the argo program in PATH, but it's not usable. argo v2.4.3 should work.")
+      warnings.warn("Cannot validate the compiled workflow. Found the argo program in PATH, but it's not usable. argo v3.1.1 should work.")
```

Comment:
nit: argo CLI v3.1.1+ should work

Reply:
+1, and we can merge the PR after the change. 

Thanks!